### PR TITLE
refactor: update exchange ui

### PR DESCRIPTION
**Pull request by dated**
## Summary

https://app.clickup.com/t/egzafa

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Comment by faustbrian** [comment]
Task linked: [CU-egzafa [Exchanges] exchange card adjustments](https://app.clickup.com/t/egzafa)

File: src/domains/exchange/components/ExchangeCard/ExchangeCard.tsx [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
 	return (
-		<Card isSelected={isSelected} onClick={onClick} actions={options}>
-			<div className="flex items-center p-2 font-semibold text-theme-primary-600">
-				<div className="mr-4">
-					<div className="w-12 h-12">
-						<Image name="ChangeNowLogo" />
-					</div>
+		<Card actions={actions} onClick={onClick} onSelect={onSelect}>
+			<div className="flex items-center space-x-4">
+				<div className="min-w-25 w-25 h-25 overflow-hidden rounded-lg">
```
Comment (ItsANameToo) [inline src/domains/exchange/components/ExchangeCard/ExchangeCard.tsx:44]: is `min-w-25` needed so it doesn't shrink? if so, `flex-shrink-0` might have worked too

File: src/domains/exchange/components/ExchangeCard/ExchangeCard.tsx [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
 	const { t } = useTranslation();
 
 	return (
 		<Card>
-			<div className="flex items-center p-2 font-semibold text-theme-primary-100 dark:text-theme-secondary-800">
-				<div className="mr-4 w-12 h-12 rounded-lg border-2 border-theme-primary-100 dark:border-theme-secondary-800" />
-
-				{t("EXCHANGE.EXCHANGE_NAME")}
+			<div className="flex items-center space-x-4">
+				<div className="h-25 w-25 rounded-lg border-2 border-theme-primary-100 dark:border-theme-secondary-800" />
```
Comment (ItsANameToo) [inline src/domains/exchange/components/ExchangeCard/ExchangeCard.tsx:21]: could probably refactor these placeholder images to a `placeholder` component so they can be reused 🤔

**Review by ItsANameToo — APPROVED** [review]

File: src/domains/exchange/components/ExchangeCard/ExchangeCard.tsx (hunk 1 above)
Comment (dated) [reply src/domains/exchange/components/ExchangeCard/ExchangeCard.tsx:44]: These lines will be replaced by the `PluginImage` component in the upcoming refactor, I'll check if `flex-shrink-0` does the job in that component 👌

**Pull request merged**